src/KeyUnit.cpp

```diff
@@ -298,7 +331,92 @@ QString KeyUnit::getKeyName( int keyCode, int modifierCode )
     else return QString("undefined key");
 }
 
+void KeyUnit::initStats()
```

Comment:
You've obviously seen similar code in a different Mudlet "item" class and taken the initiative to replicate the _stats_ reporting stuff. Kudos for doing it, though we may want to rethink this if we get around to doing something like this for all of "Buttons/Menus/Toolbars; Keys, Aliases, Timers, Triggers..." err "Scripts" __and__ "Variables"... :open_mouth: